readme.md

```diff
@@ -126,7 +126,7 @@ const Printer = require('lighthouse/lighthouse-cli/printer');
 function launchChromeAndRunLighthouse(url, flags, config) {
   const launcher = new ChromeLauncher({port: 9222, autoSelectChrome: true});
 
-  launcher.isDebuggerReady()
+  return launcher.isDebuggerReady()
```

Comment:
did this one :)
https://github.com/GoogleChrome/lighthouse/commit/44c6fb0ae10b616be6666e34f65171dde48fe3ca.